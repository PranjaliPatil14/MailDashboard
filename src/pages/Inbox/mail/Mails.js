import React from "react";
import "./mail.css";
import MailHeader from "./MailHeader";

const Mails = () => {
  return (
    <div className="inbox-mails">
      <MailHeader />
      <div className="mails-content" />
    </div>
  );
};

export default Mails;
