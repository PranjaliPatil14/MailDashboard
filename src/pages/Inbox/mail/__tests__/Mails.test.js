import React from "react";
import { render } from "@testing-library/react";
import Mails from "../Mails";

describe("Mails", () => {
  test("should render Mails", () => {
    const { asFragment } = render(<Mails />);

    expect(asFragment()).toMatchSnapshot();
  });
});
