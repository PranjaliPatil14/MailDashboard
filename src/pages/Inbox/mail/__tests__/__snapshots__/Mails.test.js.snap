// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mails should render Mails 1`] = `
<DocumentFragment>
  <div
    class="inbox-mails"
  >
    <div
      class="mails-header"
    >
      <div
        class="mails-header-title"
      >
        <h2>
          Inbox
        </h2>
        <div
          class="mail-search"
        >
          <input
            placeholder="Search email"
            type="search"
          />
          <button
            class="button "
            type="submit"
          >
            Search
          </button>
        </div>
      </div>
      <div
        class="mail-operations"
      >
        <div
          class="mail-actions"
        >
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="23 4 23 10 17 10"
              />
              <polyline
                points="1 20 1 14 7 14"
              />
              <path
                d="M3.51 9a9 9 0 0 1 14.85-3.36L23 10M1 14l4.64 4.36A9 9 0 0 0 20.49 15"
              />
            </svg>
             Refresh
          </button>
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M1 12s4-8 11-8 11 8 11 8-4 8-11 8-11-8-11-8z"
              />
              <circle
                cx="12"
                cy="12"
                r="3"
              />
            </svg>
          </button>
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <circle
                cx="12"
                cy="12"
                r="10"
              />
              <line
                x1="12"
                x2="12"
                y1="8"
                y2="12"
              />
              <line
                x1="12"
                x2="12.01"
                y1="16"
                y2="16"
              />
            </svg>
          </button>
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="3 6 5 6 21 6"
              />
              <path
                d="M19 6v14a2 2 0 0 1-2 2H7a2 2 0 0 1-2-2V6m3 0V4a2 2 0 0 1 2-2h4a2 2 0 0 1 2 2v2"
              />
              <line
                x1="10"
                x2="10"
                y1="11"
                y2="17"
              />
              <line
                x1="14"
                x2="14"
                y1="11"
                y2="17"
              />
            </svg>
          </button>
        </div>
        <div
          class="mail-navigation"
        >
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="19"
                x2="5"
                y1="12"
                y2="12"
              />
              <polyline
                points="12 19 5 12 12 5"
              />
            </svg>
          </button>
          <button
            class="button mail-action"
            type="submit"
          >
            <svg
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="5"
                x2="19"
                y1="12"
                y2="12"
              />
              <polyline
                points="12 5 19 12 12 19"
              />
            </svg>
          </button>
        </div>
      </div>
    </div>
    <div
      class="mails-content"
    />
  </div>
</DocumentFragment>
`;
